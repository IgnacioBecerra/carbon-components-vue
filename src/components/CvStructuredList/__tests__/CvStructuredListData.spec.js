import { shallowMount } from '@vue/test-utils';
import { CvStructuredListData } from '../';

const DefaultId = 'test-3';

describe('CvStructuredListData', () => {
  it('should render correctly when not wrapping text', async () => {
    const propsData = { id: DefaultId, noWrap: true };
    const wrapper = shallowMount(CvStructuredListData, { propsData });

    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render correctly when wrapping text', async () => {
    const propsData = { id: DefaultId, noWrap: false };
    const wrapper = shallowMount(CvStructuredListData, { propsData });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
