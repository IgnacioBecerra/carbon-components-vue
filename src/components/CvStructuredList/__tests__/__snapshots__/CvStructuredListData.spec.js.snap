// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CvStructuredListData should render correctly when not wrapping text 1`] = `<div class="cv-structured-list-data bx--structured-list-td bx--structured-list-content--nowrap" id="test-3"></div>`;

exports[`CvStructuredListData should render correctly when wrapping text 1`] = `<div class="cv-structured-list-data bx--structured-list-td" id="test-3"></div>`;
